### PR TITLE
Add React Native FAQ.

### PR DESCRIPTION
**Pull request by thchia**
**What**:
Add some documentation regarding usage with React Native.

**Why**:
First step to tackling a long(ish) standing [issue](https://github.com/kentcdodds/react-testing-library/issues/22) regarding this.

**How**:
Added an FAQ point discussing how one might go about using this library with React Native. Main credit to @jlongster [here](https://github.com/kentcdodds/react-testing-library/issues/22#issuecomment-423395510). I just corroborated the findings 🙂 

**Checklist**:

- [x] Documentation
- [ ] Tests
- [x] Ready to be merged
      
- [x] Added myself to contributors table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,34 +125,35 @@ test('Fetch makes an API call and displays the greeting when load-greeting is cl
 ## Table of Contents
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
+
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Installation](#installation)
```
Comment (kentcdodds) [inline README.md]: I'm guessing this has something to do with you having an old version of prettier. Could you make sure that you have the latest version?

Comment (thchia) [reply README.md]: Sure, I'll get this done in the next few hours.

**Comment by jlongster** [comment]
Thanks for going through the work to add it to the docs!

**Pull request merged**